beluga_amcl/src/amcl_nodelet.cpp

```diff
@@ -296,32 +267,29 @@ void AmclNodelet::map_timer_callback(const ros::TimerEvent &)
   map_timer_.stop();
 }
 
-bool AmclNodelet::set_map_callback(
-  nav_msgs::SetMap::Request & request,
-  nav_msgs::SetMap::Response & response)
-{
+bool AmclNodelet::set_map_callback(nav_msgs::SetMap::Request& request, nav_msgs::SetMap::Response& response) {
   std::lock_guard<std::mutex> lock(mutex_);
   NODELET_INFO("A new map has been requested to be set");
 
   if (!particle_filter_) {
     NODELET_WARN(
-      "Ignoring set map request because the "
-      "particle filter has not been initialized");
-    response.success = false;
+        "Ignoring set map request because the "
+        "particle filter has not been initialized");
+    response.success = 0U;
```

Comment:
@nahueespinosa why the type change? Same below.

Reply:
It is actually an implicit conversion from bool to unsigned char:
```
error: implicit conversion bool -> 'nav_msgs::SetMapResponse_::_success_type' (aka 'unsigned char') [readability-implicit-bool-conversion,-warnings-as-errors]
```

This was clang-tidy auto-fixer. Tried to make it "better" in https://github.com/Ekumen-OS/beluga/pull/236/commits/f91d577638967da94eca8afead823fc111ab3e83.